frontend/repositories/example/apiDocumentRepository.ts

```diff
@@ -15,6 +15,12 @@ export class APIExampleRepository implements ExampleRepository {
     return plainToInstance(ExampleItemList, response.data)
   }
 
+  async articleIds(projectId: string): Promise<Array<string>> {
+    const url = `/projects/${projectId}/article_ids?limit=999999=&offset=0`
```

Comment:
I think rather than setting the limit staticly like this, its better to set the limit as the param and make 99999 a default parameter.. what do you think?

Reply:
I agree, I think it is a better approach indeed XD I will change it into a param